src/chttpd/src/chttpd_db.erl

```diff
@@ -522,6 +522,7 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_purge">>]}=Req, Db) ->
         false -> throw({bad_request, "Exceeded maximum number of revisions."});
         true -> ok
     end,
+    couch_stats:increment_counter([couchdb, document_purges], length(IdsRevs2)),
```

Comment:
@jiangphcn : If I understand correctly,  the `document_purges` counter records how many revisions were actually purged. This would be equal to the length of all the revisions. 

If my assumption is correct, what happens if the actual call to `fabric:purge_docs` fails the following line?